Procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: bin/rails server
+css: bin/rails tailwindcss:watch
```

Comment:
If we are not using tailwind any more we might want to remove this.

Reply:
Agree!